pkg/manifests/manifests.go

```diff
@@ -2054,3 +2092,40 @@ func NewSecurityContextConstraints(manifest io.Reader) (*securityv1.SecurityCont
 
 	return &s, nil
 }
+
+func (f *Factory) TelemeterConfigmapHash(caBundleCM *v1.ConfigMap) (*v1.ConfigMap, error) {
+	// Copy the CA Bundle ConfigMap to new CM but change name to append the hash.
+	// ca-bundle.crt is the key that.
+
+	data := make(map[string]string)
+	r := newErrMapReader(data)
+
+	for k, v := range caBundleCM.Data {
+		data[k] = v
+	}
+
+	var caBundle = r.value("ca-bundle.crt")
+	if r.Error() != nil {
+		// TODO: Should we reuturn a NotExists error here instead, so we can check against that and continue.
```

Comment:
agreed on the TODO (see comment below) :+1: I think we don't have to use errors here for control flow, just a native map as mentioned above.

Reply:
Yep, when I went to do the todo I noticed as well :) Copy pasted it from the prom-adapter. ;)